bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/altair/InstAltair.java

```diff
@@ -362,16 +357,16 @@ public ISysConfig getSysConfig() {
     private static final Collection<GuideProbe> ANTI_GUIDERS    =
             GuideProbeUtil.instance.createCollection(PwfsGuideProbe.pwfs1, PwfsGuideProbe.pwfs2);
 
-    private static Set<GuideProbe> allMinus(GuideProbe guideProbe) {
+    private static Set<GuideProbe> allMinus(final ImList<GuideProbe> guideProbes) {
         final Set<GuideProbe> guiders = new TreeSet<>(GuideProbe.KeyComparator.instance);
         guiders.addAll(GuideProbeMap.instance.values());
-        guiders.remove(guideProbe);
+        guideProbes.foreach(guiders::remove);
         return guiders;
     }
 
     public Collection<GuideProbe> getConsumedGuideProbes() {
         if (getGuideStarType() == GuideStarType.LGS) {
-            return allMinus(getMode().guider());
+            return allMinus(getMode().guiders());
```

Comment:
Agreed. This is incredibly confusing and took me awhile to figure out what was happening / needed to be done.